src/common/vfs.h

```diff
@@ -126,7 +126,7 @@ class OCSYNC_EXPORT Vfs : public QObject
 
 public:
     explicit Vfs(QObject* parent = nullptr);
-    virtual ~Vfs();
```

Comment:
@mgallien Why is `virtual` removed? You don't want to call a base class' destructor when deleting an instance of a derived class?

Reply:
the fact that this is inheriting QObject that already has a destructor makes it an override of the destructor of the base class.
that has two consequences:
- we are overriding a method from base class we should add `override`
- the destructor will be made `virtual` by definition because it overrides an already virtual method